lib/std/debug.zig

```diff
@@ -288,6 +288,12 @@ pub fn dumpHexFallible(bytes: []const u8) !void {
     }
 }
 
+pub const StackTraceMode = enum {
```

Comment:
@andrewrk are you fine with this being here? A bit unclear from your comment if you meant for the enum itself to be declared inside `std.Options` too.